src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/HomeOfficeCaseNotificationsHandler.java

```diff
@@ -86,11 +88,33 @@ public boolean canHandle(
                 .getCaseDetails()
                 .getCaseData();
 
+        final String homeOfficeSearchStatus = asylumCaseWithHomeOfficeData.read(HOME_OFFICE_SEARCH_STATUS, String.class)
+            .orElse("");
+
+        final YesOrNo homeOfficeNotificationsEligible = asylumCaseWithHomeOfficeData.read(HOME_OFFICE_NOTIFICATIONS_ELIGIBLE, YesOrNo.class)
+            .orElse(YesOrNo.NO);
 
         if (asylumCaseWithHomeOfficeData.read(APPELLANT_IN_UK, YesOrNo.class).map(
             value -> value.equals(YesOrNo.YES)).orElse(true)) {
 
-            asylumCaseWithHomeOfficeData = homeOfficeApi.call(callback);
+            if ("SUCCESS".equalsIgnoreCase(homeOfficeSearchStatus)
+                && homeOfficeNotificationsEligible == YesOrNo.YES) {
+
```

Comment:
Please add info log with caseId, Home Office number, that we are about to send HO notification

Reply:
log entry added